pandas/core/style.py

```diff
@@ -255,6 +272,75 @@ def _translate(self):
                     precision=precision, table_styles=table_styles,
                     caption=caption, table_attributes=self.table_attributes)
 
+    def format(self, formatter, subset=None):
+        '''
+        Format the text display value of cells.
+
+        .. versionadded:: 0.18.0
+
```

Comment:
yep, its an ``.apply`` like function (and it returns the entire ``Styler`` so can continue to chain), so you would do:

``df.style.select_dtypes(include=['float']).format(....).select_dtypes(include=['datetime']).format(.....)``